### DISCUSSION TITLE
mcuboot : XIP is it possible to configure both slot 0 and slot 1 in external flash

### DISCUSSION THREAD

**Discussion by DineshVaghela** [mcu-tools/mcuboot#2144] [Q&A]
i would like to use active slots and both primary and secondary image is in external memory.
is it possible to configure ? 
Note : Internal image does not have enough space need both slot in external memory 

mcuboot will check active slot in external memory and jump to appropriate image

**Comment by nordicjm** [comment]
No reason why not, directXIP mode is probably best for that, but your device and drivers would need to support XIP from that device

**Reply by dineshvaghela-eaton** [reply]
Now , i am able to jump to active slots in external flash using this configuration

CONFIG_BOOT_VALIDATE_SLOT0=y
CONFIG_BOOT_DIRECT_XIP=y
CONFIG_STM32_MEMMAP=y
CONFIG_STM32_MEMMAP_EXT_FLASH_BASE_ADDRESS=0x90000000
